pkg/isb/jetstream/writer_test.go

```diff
@@ -165,6 +165,9 @@ func TestJetStreamBufferWriterBufferFull(t *testing.T) {
 	_, errs := jw.Write(ctx, messages)
 	assert.Equal(t, len(errs), 10)
 
+	// wait 2 seconds so that the buffer full check is done.
+	time.Sleep(2 * time.Second)
```

Comment:
adding this wait won't have any benefit. the assertion has already happened.

Reply:
Adding this is to make sure the isFull check has been run.

Reply:
Which is for line 172.

Reply:
Updated test case.